ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/service/support/ArticleParser.java

```diff
@@ -31,14 +31,16 @@
     private static final String ORDER_FOLDER_TAG = "sling:OrderedFolder";
     private static final String PAGE_TAG = "cq:Page";
 
+    private static final String ARTICLE_BASE_URL = BASE_URL + "/content/experience-fragments/questions_about_god";
+
     //region Public Executor
     /**
      * This executes the parsing of the local JsonObject.
      *
      * @return return a list of {@link Article}
      */
     public static List<Article> parse(@NonNull final JSONObject articleJSON) {
-        return jsonObjectHandler(articleJSON);
+        return jsonObjectHandler(articleJSON, new ArrayList<>());
```

Comment:
it's probably better to not use a `List` at all, but instead pass in a `Uri` object.

Reply:
The initial `Uri` object can be the `AemImport` base url